packages/api/package.json

```diff
@@ -33,18 +34,25 @@
     "decompress-tarbz2": "^4.1.1",
     "decompress-tarxz": "^3.0.0",
     "express": "^4.17.1",
+    "express-jwt": "^5.3.1",
     "express-pino-logger": "^4.0.0",
+    "express-session": "^1.17.0",
+    "jsonwebtoken": "^8.5.1",
+    "keycloak-connect": "^9.0.0",
     "lodash": "^4.17.15",
     "mongodb": "^3.5.0",
     "ms": "^2.1.2",
     "multer": "^1.4.2",
     "mvdir": "^1.0.5",
     "nconf": "^0.10.0",
+    "oidc": "^0.6.4",
```

Comment:
any specific reason to use `oidc` ?

Reply:
:+1:  thanks, good catch.  I was just trying out `oidc`.  It worked but `express-jwt` is closer to what we needed.